operator/deploy/role.yaml

```
@@ -272,3 +272,13 @@ rules:
   verbs:
   - get
   - list
+- apiGroups: ["iter8.tools"]
+  resources:
+  - experiments
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - watch
```

There are times when the operator is not able to grant create, delete, and patch permissions. Use the placeholders here (as we do in other apiGroups) so only the correct verbs are used:
```suggestion
  - ${OPERATOR_ROLE_CREATE}
  - ${OPERATOR_ROLE_DELETE}
  - get
  - list
  - ${OPERATOR_ROLE_PATCH}
  - watch
```

---

ah, right, I didn't see those variables.
It makes sense.